### PR TITLE
feat(layout): add qovery admin banner

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-1201

---

## PR Checklist

- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] This PR introduces new store changes
- [ ] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`
- [x] I made sure the code is type safe (no any)

### CODE REVIEW

**Comment by RemiBonnet** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery []() preview cc1de7de-94e5-40f8-8e45-bc8986d9dfec | [storybook](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/cc1de7de-94e5-40f8-8e45-bc8986d9dfec/services/general) |
 /qovery []() preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery []() preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> You are going to thank you later if it contains tests

**Comment by RemiBonnet** [comment]
[Qovery Preview](https://www.qovery.com)
---

A preview environment was automatically created via Qovery.
Click on the link below to follow its deployment and use it.
:point_right: [\[PR\] my-env - feat(layout): add qovery admin banner - 2024-07-17T08:47:01Z](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/e773b890-e6b2-4406-93a9-71f9a7ab6d73/environment/610d7d88-5cb4-4aa5-99f7-7dc90863eb89/services/general)

 Another comment will be posted when deployments are finished

**Comment by RemiBonnet** [comment]
[Qovery Preview](https://www.qovery.com)
---

Your preview environment failed to be deployed !
Click on the link below to check the deployment logs.
:point_right: [\[PR\] my-env - feat(layout): add qovery admin banner - 2024-07-17T08:47:01Z](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/e773b890-e6b2-4406-93a9-71f9a7ab6d73/environment/610d7d88-5cb4-4aa5-99f7-7dc90863eb89/logs)

File: libs/pages/layout/src/lib/ui/layout-page/layout-page.tsx [hunk 1]
```diff
@@ -73,9 +75,24 @@ export function LayoutPage(props: PropsWithChildren<LayoutPageProps>) {
...
   const clusterCredentialError = Boolean(!matchLogInfraRoute && invalidCluster)
 
+  // Display Qovery admin if we don't have the organization in the token
+  const displayQoveryAdminBanner = useMemo(() => {
+    if (isQoveryAdminUser) {
+      const checkIfUserHasOrganization = roles.some((org) => org.includes(organizationId)) ?? true
+      return !checkIfUserHasOrganization
+    }
+    return false
+  }, [roles, organizationId])
```
Comment (ctjhoa) [inline libs/pages/layout/src/lib/ui/layout-page/layout-page.tsx:85]:
```suggestion
  }, [roles, organizationId, isQoveryAdminUser])
```

**Review by ctjhoa — CHANGES_REQUESTED** [review]

File: libs/pages/layout/src/lib/ui/layout-page/layout-page.tsx (hunk 1 above)
Comment (RemiBonnet) [reply libs/pages/layout/src/lib/ui/layout-page/layout-page.tsx]: https://github.com/Qovery/console/pull/1491/commits/868192f4e643a27d44f456d7813880ce9b93c58e

**Review by ctjhoa — APPROVED** [review]

**Pull request merged**